bridges/HashnodeBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+class HashnodeBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'liamka';
+	const NAME = 'Hashnode';
+	const URI = 'https://hashnode.com';
+	const URI_ITEM = 'https://hashnode.com';
+	const CACHE_TIMEOUT = 3600; // 1hr
+	const DESCRIPTION = 'See trending or latest posts in Hashnode community.';
+	const PARAMETERS = array();
+	const LATEST_POSTS = 'https://hashnode.com/api/stories/recent?page=';
+
+	public function collectData(){
+		$url = self::URI . '/' . $this->getInput('sort');
+
+		$html = getSimpleHTMLDOM($url)
+			or returnServerError('Error while downloading the website content');
+
+		$this->items = array();
+		$time = time();
+		for ($i = 0; $i < 5; $i++) {
+			$url = self::LATEST_POSTS . $i;
+			$content = file_get_contents($url);
+			$array = json_decode($content, true);
+
+			if($array['posts'] != null) {
+				foreach($array['posts'] as $post) {
+					$item = array();
+					$item['title'] = $post['title'];
+					$item['content'] = $post['brief'];
```

Comment:
Use [`nl2br()`](https://www.php.net/manual/en/function.nl2br.php) to inserts HTML `<br>` tags after newlines (`\n`) and use `htmlspecialchars()` to stop code examples from being treated as HTML.
```suggestion
					$item['content'] = nl2br(htmlspecialchars($post['brief']));
```